src/test/java/uk/gov/pay/directdebit/events/api/DirectDebitEventsPaginationTest.java

```diff
@@ -38,43 +37,42 @@ public void testWithAllSearchParameters() {
                 .page(2)
                 .afterDate(ZonedDateTime.of(2018, 6, 29, 8, 0, 0, 0, ZoneId.of("UTC")))
                 .beforeDate(ZonedDateTime.of(2018, 6, 29, 9, 0, 0, 0, ZoneId.of("UTC")))
-                .mandateId(1234L)
-                .transactionId(5678L)
+                .mandateExternalId("1234L")
+                .transactionExternalId("5678L")
```

Comment:
These look like strings that used to be longs…

(It’s fine because they’re basically arbitrary.)

Reply:
yeah, the strings are arbitrary... as long as they're strings :)